### ISSUE TITLE
Tindie stock?

### ISSUE THREAD

**Issue by asmagill** [MickMake/Project-PiProjector#2]
I've been checking off and on, but your status for both revisions of the board at Tindie has been out of stock for some time... are you expecting to restock at some point?

If not, would it be possible for you to publish the PCB sources or Gerber files, for any version even if it's not the latest and greatest, here so we can have our own boards made?

**Comment by Compy** [comment]
@MickMake Any idea on whether or not we can get a resolution to this? It would definitely help and save me time from laying out a new breakout board!

**Comment by GMDII** [comment]
@MickMake If you are not going to restock these boards can you please upload gerbers so we can order our own? Thanks, just got 2 DLP's for a project.

**Comment by asmagill** [comment]
FWIW, I came across https://www.pcbway.com/project/shareproject/Pi_Zero_to_TI_Pico_Projector_Board.html while looking for something else, but I haven't tried it myself, so caveat emptor.

I ended up wiring a board up myself by hand that has been running atop a Pi 3 for a few months now. Been planning to do a write up about it with pictures, but haven't had the time. I was able to implement it with 2 GPIO pins free for other things (in addition to i2c which is shared).

**Comment by stephanebruckert** [comment]
> FWIW, I came across https://www.pcbway.com/project/shareproject/Pi_Zero_to_TI_Pico_Projector_Board.html while looking for something else, but I haven't tried it myself, so caveat emptor.

did you or anyone else have a chance to try this?

Edit: anyway I've ordered it, so I'll let you all know

**Comment by GMDII** [comment]
I have not looked into this. have not had time.

**Comment by stephanebruckert** [comment]
So the board at https://www.pcbway.com/project/shareproject/Pi_Zero_to_TI_Pico_Projector_Board.html was useful. It works! It fixed the shaking screen issue I had while using cables and no board.

The board was created by Sean Hodgins who wrote https://www.instructables.com/Face-Changing-Projection-Mask-Be-Anything/, which means the tutorial must be followed:
 - 3 headers must be soldered onto the board so it connects to the PI and the projector
 - a jumper must be soldered as seen here https://content.instructables.com/ORIG/FNZ/UKDH/K29B47XK/FNZUKDHK29B47XK.jpg?auto=webp&frame=1&fit=bounds&md=676ef04c29a78de55952c19696ee2d65
 -  also the pin 43 must be bent

I'm thinking of writing a full tutorial but until I haven't, feel free to ask any question and I will try to answer

**Comment by tomtechie** [comment]
@stephanebruckert 
I ended up here exactly because of the project u linked above.
Also using https://www.pcbway.com/project/shareproject/Pi_Zero_to_TI_Pico_Projector_Board.html and it works when following the instructions u posted.

I only have 1 issue that I can't really figure out and it seems like all forums about that issue has been closed. 

I can't get the raspberry pi zero to show any image on the DLP2000.

I've followed this guide: http://frederickvandenbosch.be/?p=2948

And also tried the configs here in this GitHub project but nothing other than the projectors default screen.

Basically seems like I get the same issue as this guy is quoting here: https://e2e.ti.com/support/dlp-products-group/dlp/f/dlp-products-forum/844615/dlpdlcr2000evm-evm-with-raspberry-pi4

```
"EVM hardware is found at 7, but the document says 3:

sudo i2cset -y 3 0x1b 0x0c 0x00 0x00 0x00 0x13 i
sudo i2cset -y 3 0x1b 0x0b 0x00 0x00 0x00 0x00 i
So it is failing:
Error: Could not open file `/dev/i2c-3' or `/dev/i2c/3': No such file or directory"

```


Can anyone please give me a helping hand? :) 
Happy to buy you some beers for helping!

**Comment by stephanebruckert** [comment]
@talving do you use a Pi 4 or a Pi Zero?

**Comment by tomtechie** [comment]
Pi Zero W

**Comment by tomtechie** [comment]
So my setup is basically the same as this guide here:
https://www.instructables.com/Face-Changing-Projection-Mask-Be-Anything/

**Comment by stephanebruckert** [comment]
Ok I think I know why. I had the same problem and found the answer in some forum.

In the [tutorial](http://frederickvandenbosch.be/?p=2948) it says `/boot/config.txt` needs:

    # Add support for software i2c on gpio pins
    dtoverlay=i2c-gpio,i2c_gpio_sda=23,i2c_gpio_scl=24,i2c_gpio_delay_us=2

but for Pi Zero we need to add `,bus=3` to that line, like this:

    # Add support for software i2c on gpio pins
    dtoverlay=i2c-gpio,i2c_gpio_sda=23,i2c_gpio_scl=24,i2c_gpio_delay_us=2,bus=3

save that and make sure you reboot

**Comment by tomtechie** [comment]
Seems like it's at least trying to do something now. 
When the raspberry boots the default screen goes black for half a second but then back to the default screen and nothing..

**Comment by tomtechie** [comment]
and I can now get information when running `sudo i2cdetect -y 3`

```
     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00:                         -- -- -- -- -- -- -- -- 
10: -- -- -- -- -- -- -- -- -- -- -- 1b -- -- -- -- 
20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
30: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
50: -- -- -- -- -- -- -- 57 -- -- -- -- -- -- -- -- 
60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
70: -- -- -- -- -- -- -- --       
```

**Comment by stephanebruckert** [comment]
I remember adding some more lines to the end of `boot/config.txt` (particularly the last 5), here is my full file:

```bash
$ cat /boot/config.txt 
# Uncomment some or all of these to enable the optional hardware interfaces
dtparam=i2c_arm=on
#dtparam=i2s=on
dtparam=spi=on

# Uncomment this to enable infrared communication.
#dtoverlay=gpio-ir,gpio_pin=17
#dtoverlay=gpio-ir-tx,gpio_pin=18

# Additional overlays and parameters are documented /boot/overlays/README

# Enable audio (loads snd_bcm2835)
dtparam=audio=on

[pi4]
# Enable DRM VC4 V3D driver on top of the dispmanx display stack
dtoverlay=vc4-fkms-v3d
max_framebuffers=2

[all]
#dtoverlay=vc4-fkms-v3d

# Add support for software i2c on gpio pins
dtoverlay=i2c-gpio,i2c_gpio_sda=23,i2c_gpio_scl=24,i2c_gpio_delay_us=2,bus=3

# DPI Video Setup
dtoverlay=dpi18
overscan_left=0
overscan_right=0
overscan_top=0
overscan_bottom=0
framebuffer_width=854
framebuffer_height=480
enable_dpi_lcd=1
display_default_lcd=1
dpi_group=2
dpi_mode=87

dpi_output_format=458773
hdmi_timings=854 0 14 4 12 480 0 2 3 9 0 0 0 60 0 32000000 3

start_x=1
gpu_mem=128
enable_uart=1
dtoverlay=w1-gpio
dtparam=i2c1=on
```

Hope it helps

**Comment by stephanebruckert** [comment]
FYI I started from this tutorial https://www.mickmake.com/post/build-a-pi-zero-w-pocket-projector-project/

The part about enabling i2c is probably important. I assume it is what `dtparam=i2c1=on` does

**Comment by tomtechie** [comment]
Will have a look at that. 
Just out of curiosity I disconnected the projector and plugged it to my TV with HDMI and it's only showing the raspberry rainbow screen. Do u know if this is normal? 
Can access the raspberry from SSH and everything.

**Comment by stephanebruckert** [comment]
>  I disconnected the projector and plugged it 

"it", the projector or raspberry? Asking because the rainbow screen is part of the DLP and the DLP doesn't have an HDMI, so I don't understand how that is possible

**Comment by tomtechie** [comment]
Sorry,
Disconnected the Projector from the raspberry pi and turned on the Raspberry while connected to HDMI to TV. And output to TV is only showing rainbow. Guessing the
```
# Add support for software i2c on gpio pins
dtoverlay=i2c-gpio,i2c_gpio_sda=23,i2c_gpio_scl=24,i2c_gpio_delay_us=2,bus=3

# DPI Video Setup
dtoverlay=dpi18
overscan_left=0
overscan_right=0
overscan_top=0
overscan_bottom=0
framebuffer_width=854
framebuffer_height=480
enable_dpi_lcd=1
display_default_lcd=1
dpi_group=2
dpi_mode=87

dpi_output_format=458773
hdmi_timings=854 0 14 4 12 480 0 2 3 9 0 0 0 60 0 32000000 3

start_x=1
gpu_mem=128
enable_uart=1
dtoverlay=w1-gpio
dtparam=i2c1=on
```

Disables HDMI output. 



On other notes the last 5 rows does not seem to solve my issue. no output to projector.. Really appreciate the help!

**Comment by kuraga** [comment]
Any news on this?
